adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/BinaryResource.kt

```diff
@@ -57,7 +62,7 @@ class BinaryResource {
         arch: Architecture?,
 
         @Parameter(name = "release_name", description = OpenApiDocs.RELASE_NAME, required = true,
-                   schema = Schema(defaultValue = "jdk-11.0.6+10", type = SchemaType.STRING)
+            schema = Schema(defaultValue = "jdk-11.0.6+10", type = SchemaType.STRING)
```

Comment:
just realized that having the version string in there is a little brittle, but one to fix for later.